src/i18n/zh.json

```diff
@@ -0,0 +1,44 @@
+{
+  "header": {
+    "builtby": "built with 🌮 by"
+  },
+  "form": {
+    "columns": "列",
+    "rows": "行",
+    "columngap": "列间距",
+    "rowgap": "行间距",
+    "codebutton": "请给我示例中的代码",
+    "project": "这个项目可以做什么？",
+    "reset": "重置布局"
+  },
+  "modal": {
+    "header": {
+      "yourcode": "您的 Grid 代码",
+      "what": "这是什么？"
+    },
+    "copy": {
+      "title": "复制以下代码：",
+      "clipboard": "复制"
+    },
+    "button": "完成"
+  },
+  "grid": {
+    "realcssunit": "必须使用正确的 CSS 单位"
+  },
+  "explain": {
+    "paragraph1": "您可以设置行和列的数字还有单位，我将为您生成一个 CSS Grid 网格布局！<strong>在方框中拖动</strong>来创建 div 放置在网格内。",
+    "paragraph2": "虽然这个项目可以为您提供一个基本的布局, 但是本项目不是对 CSS 网格功能的全面介绍。这只是一种快速使用 CSS Grid 网格布局功能的方法。",
+    "paragraph3": "我注意到很多人没有使用 Gird，因为他们觉得这个太复杂了，难以理解。 Grid 可以做很多事情，而这个小的生成器只涉及到了一小部分功能。 这样做的目的是让人们能够快速上手并创建更多有趣的布局。",
```

Comment:
Is `Gird` here a typo?

Reply:
Yes, I am careless, thank you very much.😃It's been solved.